src/filter/addons/filter-item-checkbox.tsx

```diff
@@ -196,6 +196,13 @@ export const FilterItemCheckbox = <T,>({
                 styleType="link"
                 type="button"
                 onClick={handleSelectClearAll}
+                aria-label={
+                    selected.length
+                        ? "Clear all selections"
+                        : `Select all in ${
+                              parentRef.current?.ariaLabel || "this group"
+                          }`
+                }
```

Comment:
on further testing, the button now is associated with the group, so would it be ok to use the default accessible label?

<img width="663" height="483" alt="Screenshot 2025-08-08 at 2 14 26 PM" src="https://github.com/user-attachments/assets/6ff8a196-7b4b-461d-824d-e6a95cf45bb5" />

Reply:
sorry for the flip flop, on Android it's not announcing the group, so the context for the buttons within each filter item is still missing